tools/mo/openvino/tools/mo/back/ReverseInputChannels.py

```diff
@@ -536,7 +475,7 @@ def find_and_replace_pattern(self, graph: Graph):
 
 class ApplyReverseChannels(BackReplacementPattern):
```

Comment:
As I understand (it is not mentioned anywhere!) this code is left here to fuse internal RIC operator for TensorFlow models with Pack-Unpack pattern. So, why can we adopt current logic by @GlebKazantaev and delete this file completely?

Reply:
You are right, I don't think we should delete it.